nixos/modules/services/ttys/kmscon.nix

```diff
@@ -32,6 +32,13 @@ in {
         default = "";
         example = "font-size=14";
       };
+
+      extraOptions = mkOption {
+        description = "Extra flags to pass to kmscon.";
+        type = types.string;
```

Comment:
This should probably be a `types.listOf types.str`, and then we space-separate the options in `ExecStart`